cluster/images/etcd/README.md

```diff
@@ -48,7 +48,7 @@ directory permissions and 0644 file permissions.
 
 For `amd64`, official `etcd` and `etcdctl` binaries are downloaded from Github
 to maintain official support.  For other architectures, `etcd` is cross-compiled
-from source. Arch-specific `busybox` images serve as base images.
+from source. Arch-specific `bullseye` images serve as base images.
```

Comment:
```suggestion
from source. Arch-specific `debian` images serve as base images.
```
Debian? `bullseye` might not be specific enough https://www.google.com/search?q=bullseye.